cmp/dock/DockContainerModel.js

```diff
@@ -25,25 +25,39 @@ export class DockContainerModel {
     /** @member {string} */
     direction;
 
+    /** @member {RenderMode} */
+    renderMode;
+
+    /** @member {RefreshMode} */
+    refreshMode;
+
     /**
      * @param {Object} [c] - DockContainerModel configuration.
      * @param {Object[]} [c.views] - DockViewModel configs to be displayed.
      * @param {string} [c.direction] - direction in which docked views build up as they are added to
      *      the container. Valid values are 'ltr', 'rtl' - the default of 'rtl' causes the first
      *      view to be docked to the bottom right of the container with each subsequent view docked
      *      to the left of view before it.
+     * @param {RenderMode} [c.renderMode] - strategy for rendering DockViews. Can be set
+     *      per-tab via `DockViewModel.renderMode`. See enum for description of supported modes.
+     * @param {RefreshMode} [c.refreshMode] - strategy for refreshing DockViews. Can be set
+     *      per-tab via `DockViewModel.refreshMode`. See enum for description of supported modes.
      */
     constructor({
         views = [],
-        direction = 'rtl'
+        direction = 'rtl',
+        renderMode = RenderMode.UNMOUNT_ON_HIDE,
```

Comment:
Why is this default different from the TabContainer default?  e.g.  renderMode = RenderMode.LAZY.
Do we have some sense that application authors would expect these things to unmount by default?